src/test/unit_tests/jest_tests/test_versioning_concurrency.test.js

```diff
@@ -193,11 +467,10 @@ describe('test versioning concurrency', () => {
 async function _upload_versions(bucket, key, number_of_versions) {
     const versions_arr = [];
     for (let i = 0; i < number_of_versions; i++) {
-        const random_data = Buffer.from(String(i));
+        const random_data = Buffer.from(String(crypto_random_string(7)));
```

Comment:
Would you please explain this small change?

Reply:
instead of creating a buffer from the number we create a random string

Reply:
Thanks, what was the reason for the change?

Reply:
it looked like a mistake